aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,69 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+    - [CheckboxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=ViKCtdnRM2CUkNZI-1)
+    - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=ViKCtdnRM2CUkNZI-1)
+    - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=ViKCtdnRM2CUkNZI-1)
+    - [TextInput](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A8&t=ViKCtdnRM2CUkNZI-1)
+    - [Layer](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=ViKCtdnRM2CUkNZI-1)  
+- **FileInput**
+  - We have added the alert icon within [FileInput in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=387%3A1967&t=ViKCtdnRM2CUkNZI-1).
+- **Data**
+  - [Data Filters now have a tooltip](https://design-system.hpe.design/templates/filtering#filtering-with-selectable-results) when hovering over the filter button.
+
+## March 2023
+
+- **Data**
+  - Refactored [Data on DataTable Customization on site](https://design-system.hpe.design/templates/datatable-customization)
+- **DateInput**
+  - Updated the heading to use text on site and Figma
+- **Learn**
+  - Added [Box Model One tutorial to the Learn page](https://design-system.hpe.design/learn/the-box-model-part-one) on site.
+- **Layer**
+  - [Layers now have overflow property](https://design-system.hpe.design/templates/filtering?q=filte) to ensure that the content does not exceed the container, making it easier for user to view all filter content.
+- **Tabs**
+  - Updated [Tabs styling](https://design-system.hpe.design/components/tabs?q=tabs) based on new brand refresh changes.
+- **Page Templates**
+  - [List + Table Figma templates](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5639%3A166208&t=IC28JVzDR8NsAZ30-1) have been completed.
+  - [Detail/Object View Figma templates](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5768%3A218145&t=IC28JVzDR8NsAZ30-1) have been completed.
+- **Brand Refresh**
+  - Brand Audit on Grommet has been completed.
+  - Older [Card examples](https://design-system.hpe.design/templates/filtering?q=filterin) are now using a Heading with the correct semantic level.
+  - New brand refresh now available through out the site! 
+  - To view details on changes made to the new theme, view [V5 release notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v5.0.0).
+  - To view details on how to migrate from HPE Theme 4.x to 5.x, please visit the [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-4.x-to-5.x).
+
+## February 2023
+
+- **Brand Refresh**
+  - Changed HPE theme to brand Metric font
+  - Conducted a brand audit on GLCP to identify and track unaligned elements to the new and updated HPE Theme.
```

Comment:
```suggestion
```
Not sure that we need this update, it seems like the people that care about this would already be informed and it doesn't relate to all consumers of the design system.